respy/tests/random_model.py

```diff
@@ -93,6 +95,9 @@ def generate_random_model(
         Number of unobserved types.
     n_type_covariates :
         Number of covariates to calculate type probabilities.
+    n_observables: list
```

Comment:
it is not the number of observables.
```suggestion
    observables: list
```